elfi/visualization/visualization.py

```diff
@@ -397,3 +397,50 @@ def plot_discrepancy(gp, parameter_names, axes=None, **kwargs):
         axes[idx].set_axis_off()
 
     return axes
+
+
+def plot_gp(gp, parameter_names, axes=None, resol=50, const=None, **kwargs):
+    """Plot pairwise relationships as a matrix with parameters vs. discrepancy.
+
+    Parameters
+    ----------
+    gp : GPyRegression, required
+    parameter_names : dict, required
+        Parameter names from model.parameters dict('parameter_name':(lower, upper), ... )`
```

Comment:
Names or bounds? If just the names, why is a dict required?

Reply:
I think parameter_names should list or tuple instead of dict

Reply:
It is a _list_.